### PR TITLE
Fix invalid "sequence" parameter in dynamic content example

### PR DESCRIPTION
**Pull request by chayleaf**
While it doesn't matter in this particular example since row and column count doesn't change, it might cause issues when generating more dynamic content

### CODE REVIEW

**Comment by rstropek** [comment]
Not sure what's wrong with the current implementation. Can you elaborate why your changes are necessary/better?

**Comment by chayleaf** [comment]
the source code says it should specify the sequence number *in the source code*, and that's also how autogenerated code works. I'm not sure what exactly sequence numbers are used for, but that's just how they need to be provided.

**Comment by rstropek** [comment]
The existing code already generates sequence numbers. The sequence number is just automatically incremented from component to component to make the code more easy to maintain. Is that what you do not like?

**Comment by chayleaf** [comment]
the sequence numbers in the existing code will increment upon each iteration, while they should stay the same across iterations

**Comment by rstropek** [comment]
Really? I don't think so. If I would build the grid manually (without `for` loop), each element would also get a new sequence number. The idea of the sample is to show that you can create a render tree using an algorithm. However, the generated structure should look the same as if it was build manually. Do you have a different understanding?

**Comment by chayleaf** [comment]
if you would build it manually *without loops*, each element would get a new sequence number, however if you used a loop, the elements would have the same sequence across iterations. If you look at the docs: https://github.com/aspnet/AspNetCore/blob/master/src/Components/Components/src/RenderTree/RenderTreeBuilder.cs

it says: `<param name="sequence">An integer that represents the position of the instruction in the source code.</param>`

and if you did something like

```cs
@for (int i = 0; i < items.Count; i++)
{
    <p>@items[i].Text</p>
}
```

the sequence number wouldn't increment with iterations

basically I want to show the users the proper way to use sequence numbers. As I said before, this example works because the iteration count is predefined, but you can't do the same for loops with different iteration count.

**Comment by rstropek** [comment]
Got you. You are absolutely right. Will merge the pull request. Thank you for pointing that out.
